setup.py

```diff
@@ -62,7 +62,7 @@
     "neural-compressor": ["neural-compressor>=2.2.0", "onnxruntime<1.15.0", "accelerate"],
     "openvino": ["openvino>=2023.3", "nncf>=2.10.0", "openvino-tokenizers[transformers]"],
     "nncf": ["nncf>=2.10.0"],
-    "ipex": ["intel-extension-for-pytorch", "transformers>=4.36.0,<4.39.0"],
+    "ipex": ["intel-extension-for-pytorch", "transformers>=4.36.0,<=4.41.2"],
```

Comment:
in the tests I see the minimum tested version is v4.38.2 so we need to update it here as well 
```suggestion
    "ipex": ["intel-extension-for-pytorch", "transformers>=4.38.2,<=4.41.2"],
```